spec/javascript/pages/browse/ListBrowser.spec.js

```diff
@@ -1,16 +1,36 @@
-import {assert} from 'chai'
```

Comment:
why isn't assert needed from chai anymore?

Reply:
I think that's because our `setup.js` file brings it in, and that setup file is explicitly included in https://github.com/rubyforgood/mutual-aid/blob/8086ba858d0f760088ed0785b4f68e706cff3121/package.json#L50